package.json

```diff
@@ -25,14 +25,14 @@
     "@types/chai": "4.2.11",
     "@types/jsforce": "^1.9.18",
     "@types/mocha": "^7.0.2",
-    "@types/node": "^13.13.2",
+    "@types/node": "^16.0.2",
```

Comment:
This dependency is also being updated in the other PR(https://github.com/salesforce/sfdx-plugin-lwc-test/pull/30/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R28)